frontend/constants/networks.ts

```diff
@@ -22,6 +24,10 @@ const NETWORKS = {
     rpcUrl: 'https://rpc.testnet.lukso.network',
     explorerURL: 'https://explorer.execution.testnet.lukso.network',
     universalGraveForwarder: '0x7e77c704be7a6bba042f66eaba0b9557a872902d',
+    previousGraveForwarders: [
+      '0x1e2f66d2dC19bB13617D1d06CBb3E8d8aa4567e3',
+      '0x9c27a05310dC8aF53B60124B244cc9d12f202cdF',
```

Comment:
you can test this change by putting this value into line 26 `universalGraveForwarder` and removing it from here, subscribe then revert the change to simulate an upgrade